source/elements/oneMKL/source/domains/dft/dft.rst

```diff
@@ -137,45 +131,48 @@ DFTs. Specifically, for any of the :math:`M` sequences,
 Recommended usage
 +++++++++++++++++
 
-The desired (batched) DFT to be computed is entirely defined by an object
-``desc`` of the :ref:`descriptor<onemkl_dft_descriptor>` class. The desired type
-of forward domain and precision are determined at ``desc``'s construction time
-by the specialization values chosen for the self-explanatory template parameters
-``prec`` (of type :ref:`onemkl_dft_enum_precision`) and ``dom`` (of type
-:ref:`onemkl_dft_enum_domain`), respectively. The transform size 
-:math:`n_1 \times n_2 \times \dots \times n_d` is also set at construction time
-as a required argument to the class constructor. Other configuration details for
-the (batched) DFT under consideration may be specified by invoking the
-:ref:`onemkl_dft_descriptor_set_value` member function of ``desc`` for every
-relevant configuration setting (*e.g.*, the number :math:`M` of sequences to
-consider in case of a batched DFT). Once configured as desired, the
-:ref:`onemkl_dft_descriptor_commit` member function of ``desc``, requiring a
-``sycl::queue`` object ``Q``, may be invoked. The successful completion of the
-latter makes ``desc`` committed to the desired (batched) DFT *as configured*, for
-the particular device and context encapsulated by ``Q``. The
-:ref:`onemkl_dft_compute_forward` (resp. :ref:`onemkl_dft_compute_backward`)
-function may then be called and provided with ``desc`` to enqueue operations
-relevant to the desired forward (resp. backward) DFT calculations with
-user-provided, device-accessible data.
+The desired DFT to be computed is entirely defined by an object ``desc`` of a
+specialization of the ``oneapi::mkl::dft::descriptor``
+:ref:`class template<onemkl_dft_descriptor>`.
+The desired floating-point format and kind of forward domain are determined by
+``desc``'s particular class, *i.e.*, by the specialization values of the
+:ref:`template parameters<onemkl_dft_descriptor_template_parameters>` ``prec``
+and ``dom`` of the ``descriptor`` class template, respectively. Once ``desc`` is
+created, the length(s) :math:`\lbrace n_1, n_2, \ldots, n_d\rbrace` (and the
+dimension :math:`d`) cannot be changed, as they are read-only parameters. Other
+configuration details for the DFT under consideration may be specified by
+invoking the appropriate
+:ref:`configuration-setting member function(s)<onemkl_dft_descriptor_set_value>`
+of ``desc`` for every relevant configuration parameter (*e.g.*, the number
+:math:`M` of sequences to consider in case of a batched DFT). Once configured
+as desired, ``desc`` must be initialized for computation by using its
+:ref:`committing member function<onemkl_dft_descriptor_commit>`, which requires
+a ``sycl::queue`` object ``Q``. The successful completion of the latter makes
```

Comment:
Nitpicking, I find it weird to name the object `Q`. There is another reference just below.
I don't think we need to name it here, if we do might as well keep the same name as the argument for the commit function: `queue`.

Reply:
Agreed, modified in 5eb493630d7db202522f130882e391143de6fe9b.